packages/html-data/bin/aria.ts

```diff
@@ -24,6 +24,13 @@ type Attribute = {
   options?: string[];
 };
 
+const overrides: Record<string, Partial<Attribute>> = {
+  "aria-label": {
+    description:
```

Comment:
why is it mauallly written?

Reply:
Because step didn't have a good one. AI didn't generate a good one either (we had the same manual override).